src/gradedarray.jl

```diff
@@ -4,11 +4,21 @@ using BlockSparseArrays:
   AbstractBlockSparseMatrix,
   AnyAbstractBlockSparseArray,
   BlockSparseArray,
+  BlockSparseMatrix,
+  BlockSparseVector,
   blocktype
 using ..GradedUnitRanges: AbstractGradedUnitRange, dual
 using LinearAlgebra: Adjoint
 using TypeParameterAccessors: similartype, unwrap_array_type
 
+const GradedArray{T,M,A,Blocks,Axes} = BlockSparseArray{
+  T,M,A,Blocks,Axes
+} where {Axes<:Tuple{AbstractGradedUnitRange,Vararg{AbstractGradedUnitRange}}}
+const GradedMatrix{T,A,Blocks,Axes} =
+  BlockSparseMatrix{T,A,Blocks,Axes} where {Axes<:Tuple{Vararg{AbstractGradedUnitRange}}}
```

Comment:
```suggestion
  GradedArray{T,2,A,Blocks,Axes}
```

Reply:
Just a bit simpler, and also more future-proof if we decide to change the definition of `GradedArray`.